### ISSUE TITLE
No training, data loader problems

### ISSUE THREAD

**Issue by ABerry057** [znxlwm/pytorch-CartoonGAN#2]
znxlwm,

The loss functions are giving me NaN values and it seems training is not happening at all. I have traced the error back to the for loop that iterates over the zipped train_loader_src and train_loader_tgt. The for loop never runs because zipping those two data loaders returns an empty list. I think the arguments train_loader_src and train_loader_tgt are empty/malformed. I don't entirely understand how data_load() works but I think that is where my ultimate problem lies. Do you have any advice?

I am new to PyTorch but this CartoonGAN is helping me learn. Thank you for sharing the code.
